ddd-koans/Introduction/About course/task.html

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
Should we consider adding a table of contents here in form of a simple list?

Reply:
good idea, will do it!

Reply:
done